mcp_sota/mcp_sota_26_07_28.html

```
@@ -0,0 +1,568 @@
+  </section>
+
+  <section>
+    <h3>TMLL  Tool Annotations</h3>
+    <pre><code class="language-python">from mcp.types import ToolAnnotations
+
+@mcp.tool(annotations=ToolAnnotations(
+    readOnlyHint=True,
+    destructiveHint=False,
+    openWorldHint=False
+))
+async def list_experiments() -> str:
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)